tools/base/fpgaconf/fpgaconf.c

```diff
@@ -237,6 +240,7 @@ int parse_args(int argc, char *argv[])
 		{"device",        required_argument, NULL, 'd'},
 		{"function",      required_argument, NULL, 'f'},
 		{"socket",        required_argument, NULL, 's'},
+		{"force",         no_argument,       NULL, 0xf},
```

Comment:
The only concern I have with this is I feel that this should be the default behavior when trying to do a PR.  For fpgainfo i feel there could be an argument made for an additional flag but when doing reconfiguration we should report if the fpga is already in use to notify the user.

Reply:
If we default to `force` during PR, then we get back to what started this PR (pull request).
Setting the `force` flag will bypass the check for the resource-in-use.

Reply:
The default behavior should be the one that doesn't cause any disruption or error. If an accelerator is in use, PR would disrupt the owning application (and possibly cause a undesired port error condition); those consequences should be accepted explicitly (by providing "-f"), not implicitly.